AiPlatform/src/V1/Scheduling.php

```diff
@@ -167,43 +157,5 @@ public function setDisableRetries($var)
         return $this;
     }
 
-    /**
-     * Optional. This is the maximum time a user will wait in the QRM queue for
-     * resources. Default is 1 day
-     *
-     * Generated from protobuf field <code>.google.protobuf.Duration max_wait_duration = 6 [(.google.api.field_behavior) = OPTIONAL];</code>
-     * @return \Google\Protobuf\Duration|null
-     */
-    public function getMaxWaitDuration()
```

Comment:
These seem to be BC breaking changes. I will need to validate whether these are intended. This PR is marked as `fix!` but there is no Breaking change justification provided.